package.json

```diff
@@ -15,20 +15,21 @@
     "clean": "gatsby clean"
   },
   "dependencies": {
-    "@emotion/react": "^11.1.5",
-    "@emotion/styled": "^11.1.5",
-    "gatsby": "^2.31.1",
-    "gatsby-plugin-emotion": "^5.4.0",
-    "gatsby-plugin-manifest": "^2.12.0",
-    "gatsby-plugin-react-helmet": "^3.10.0",
-    "lodash": "^4.17.20",
-    "react": "^16.13.1",
-    "react-anchor-link-smooth-scroll": "^1.0.12",
-    "react-dom": "^16.13.1",
-    "react-helmet": "^6.1.0",
-    "theme-ui": "^0.3.5"
+    "@emotion/react": "11.1.5",
```

Comment:
best not to include these changes in the diff. were they done purposefully? or just some automatic formatting?

Reply:
Yes, these changes were done on purpose. I had some dependency problems in the latest versions, so I fixated them. I did reverted this in the last batch of commits.